apis/nucleus/src/components/listbox/ListBox.jsx

```diff
@@ -218,10 +218,18 @@ export default function ListBox({
   const isVertical = listLayout !== 'horizontal';
   const count = layout.qListObject.qSize.qcy;
 
-  const getAccumulatedHeight = (ps) => ps.reduce((acc, p) => acc + p.qArea.qHeight, 0);
-
-  const listCount = pages && pages.length && calculatePagesHeight ? getAccumulatedHeight(pages) : count;
+  const PAGE_MAX_HEIGHT = 100;
+
+  const getCalculatedHeight = (ps) => {
+    // If values have been filtered in the currently loaded page, we want to
+    // prevent rendering empty rows by assigning the actual number of items to render
+    // since count (qcy) does not reflect this in DQ mode currently.
+    const h = ps[0].qArea.qHeight;
```

Comment:
why are you checking only ps[0] and not ps[1] (when it exists)

Reply:
Yes, we should probably apply this to unlimited pages. My original guess/assumption was that only the first page (index == 0) will be affected when filtering applies. But we could of course have more than 100 (or `max_items_per_page`) of filtered values, which suggests more than one page is affected. Also, this should then affect the last page rather than the first.

Let me push a change and hear what you think about it.

Reply:
My current understaning (that I am unsure about) is that you can have one or two pages.
For two pages ps[1] is the last requeted page and ps[0] the previous requested page (requested by react-window-infinite-loader).
That the requested pages can be overlapping or have space beween them.
And that any of the pages could start before the other.